server/src/lib.rs

```diff
@@ -67,10 +71,49 @@ pub async fn run(
     address: String,
     database_path: String,
     storage_path: String,
+    local_distributed_address: String,
+    all_servers_address: Vec<String>,
 ) -> anyhow::Result<()> {
     let listener = TcpListener::bind(&address).await?;
     let local_storage = Arc::new(DefaultEngine::new(&database_path, &storage_path));
-    let engine = Arc::new(DistributedEngine::new(address.clone(), local_storage));
+    local_storage.init();
+    build_hash_ring(all_servers_address.clone());
+    let service = RPCService::new(local_storage.clone());
+    let local = local_distributed_address.clone();
+    tokio::spawn(async move {
+        tonic::transport::Server::builder()
+            .add_service(engine_rpc::new_manager_service(service))
+            .serve((&local).parse().unwrap())
+            .await
+    });
+    let engine = Arc::new(DistributedEngine::new(
+        local_distributed_address.clone(),
+        local_storage.clone(),
+    ));
+    for value in all_servers_address.clone().into_iter() {
+        engine.add_connection(value, None);
```

Comment:
why need to add None here and then update in https://github.com/labring/sealfs/pull/37/files#diff-64aa9b484d4b3759cc0dd469329aa5a6bab87366c6c55eb61d9a1d1129201fcdR109

Reply:
deleted